src/java.base/share/classes/java/lang/reflect/Modifier.java

```diff
@@ -221,6 +231,22 @@ public static boolean isStrict(int mod) {
      * {@code toString} with the appropriate mask from a method like
      * {@link #constructorModifiers} or {@link #methodModifiers}.
      *
+     * @apiNote
+     * To make a high-fidelity representation of the Java source
+     * modifiers of a class or member, source-level modifiers that do
+     * not <em>not</em> have a constant in the this class should be
+     * included and ordered consistent with the full recommended
```

Comment:
```suggestion
     * included and ordered consistently with the full recommended
```

Reply:
Yes, "ordered consistently with" or "with order consistent with", I think both will work here.

Reply:
I think, Joe might be considering "consistent" to be [flat](https://en.wikipedia.org/wiki/Flat_adverb).